nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 10 Line 4_
```python
# Add records to the users table
users.insert({'name': 'Braden', 'email': 'b@example.com', 'year_started': 2018})
users.insert({'name': 'Alma', 'email': 'a@example.com', 'year_started': 2019})
users.insert({'name': 'Charlie', 'email': 'c@example.com', 'year_started': 2018})
```
It's unusual to just show the last inserted object -- or at least without mentioning that. Maybe instead for the last row say `user = ...` and then add a new cell underneath with just `user` in it, and a explanation saying what this now contains.